test/statistics.jl

```diff
@@ -55,9 +55,9 @@ end
 @test mean([1,2,3]) == 2.
 @test mean([0 1 2; 4 5 6], 1) == [2.  3.  4.]
 @test mean([1 2 3; 4 5 6], 1) == [2.5 3.5 4.5]
-@test mean(i->i+1, 0:2) === 2.
-@test mean(isodd, [3]) === 1.
-@test mean(x->3x, (1,1)) === 2.
+@test mean(i->i+1, 0:2) == 2.
+@test mean(isodd, [3]) == 1.
+@test mean(x->3x, (1,1)) == 2.
```

Comment:
Yeah, it was passing before. I'll change it back to `===`. Should I wait for the CI to catch up a bit?